data/bactrian.txt

```diff
@@ -0,0 +1,379 @@
+			`	Lieutenant Paris is waiting for you outside of your ship, and hands you another 100,000 credits when you walk up to him.`
+			`	"Thanks for all the help, Captain <last>. We couldn't have done it without you."`
+			`	Paris looks back to the Star Queen, and you can overhear talk of who survived and who died. "Sarah, Nicole, Abby..." the list of names goes on for about two dozen more, most of which female sounding. "...and Benjamin are all either missing or were found dead on board. The surviving crew tells us that some were killed when they were captured or thrown out the airlock for struggling, but most of the others were either killed or dragged off at the facility."`
+			`	The entire time you noticed Paris holding back tears, his face becoming more and more strained with each passing name. After they are done listing the dead or missing, he talks to you in a rather strained voice. "I want you to return to Arneb and take out every single one of those pirates that are left. Especially the Bactrian. We try to keep Bactrians out of unworthy hands, so whenever a pirate gets ahold of one we make sure we destroy it immediately. Seeing as how we're going to be busy here for a while, you need to destroy it."`
+			choice
+				`	"I'll make sure it gets done, Lieutenant"`
+					goto accept
+				`	"I'm sorry, but I don't have the power to be able to defeat a Bactrian. You're going to need to do it yourself."`
+			
+			`	"I see... Good luck on your travels, Captain <last>." Lieutenant Paris starts walking over to the Star Queen, leaving you alone next to your ship.`
+				decline
+			
+			label accept
+			`	"Thank you, <first>." Lieutenant Paris starts walking over to the Star Queen, leaving you alone next to your ship.`
+				accept
+	
+	on visit
+		dialog "You landed on <planet>, but you didn't kill the Bactrian! Return to Arneb and make sure that the Bactrian is destroyed and forever out of pirate hands."
+	
+	to complete
+		has "Bactrian Unlock [2]: Pirate Bactrian: done"
+	
+	on complete
+		set "license: City-ship"
+		conversation
+			`When you return to <planet>, the Star Queen is no longer on the landing pad and things seem to be running normally. Shortly after landing, you find Lieutenant Paris at the same restaurant that he first found you. Before saying a word to you, he hands you a license.`
+			`	"Thank you. I've said it before, but there's really no way I can thank you enough."`
+			`	You look down at the license that Paris just handed you and notice that it's a city-ship license, one that will allow you to purchase the Bactrian. "I think it's only fair that you be able to purchase the Bactrian after having to fight one to the death. It's really the least I could do for you."`
```

Comment:
This is the same type of language as "Alondo thought it likely that..."
Correct as is.